engine/access/rest/websockets/controller.go

```diff
@@ -394,7 +425,24 @@ func (c *Controller) handleMessage(ctx context.Context, message json.RawMessage)
 	return nil
 }
 
+// handleSubscribe processes a subscription request.
+//
+// Expected error returns during normal operations:
+//   - ErrMaxSubscriptionsReached: if the maximum number of active subscriptions per connection is exceeded.
+//   - context.Canceled: if the operation is canceled, during an unsubscribe action.
```

Comment:
it looks like `context.Canceled` is handled by closing without an error. am I missing a case?